packages/web-sockets/.eslintrc.json

```
@@ -0,0 +1,19 @@
+{
+  "env": {
+    "commonjs": true,
+    "es2021": true,
+    "node": true
```

Can you add `jest` here?
```suggestion
    "node": true,
    "jest": true
```